templates/index.html.twig

```diff
@@ -75,18 +65,18 @@
                     <div class="fr-card">
```

Comment:
Concernant les cartes (j'espère mettre ce commentaire au bon endroit)

C'est devenu sec côté maquette, Anne-So vient de proposer une 2e version en bas de la 1ère avec des cartes cliquables avec une flèche au lieu du bouton call to action

https://www.figma.com/file/YuT6Uh4wxe90U8LaPLlDzO?node-id=2812:140441&mode=design#519469569

Reply:
@Lealefoulon Est-ce que tu penses reprendre les cartes dans cette PR, ou dans une PR suivante ?